packable/packable/src/packable/string.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+extern crate alloc;
+
+use crate::{
+    error::{UnpackError, UnpackErrorExt},
+    packer::Packer,
+    prefix::UnpackPrefixError,
+    unpacker::Unpacker,
+    Packable,
+};
+
+use alloc::string::{FromUtf8Error, String};
+use alloc::vec::Vec;
```

Comment:
Merge

Reply:
you are the one who can merge the PR Thibault :trollface: